Library/Homebrew/utils.rb

```diff
@@ -498,11 +498,19 @@ def pretty_ratelimit_reset(reset)
 
   class AuthenticationFailedError < Error
     def initialize(error)
-      super <<-EOS.undent
-        GitHub #{error}
-        HOMEBREW_GITHUB_API_TOKEN may be invalid or expired, check:
+      message = "Github #{error}\n"
```

Comment:
In this case I think it's not very enlightening:

```
irb(main):023:0> headers = {"Accept" => "application/vnd.github.v3+json", "Authorization" => "token 12345"}
=> {"Accept"=>"application/vnd.github.v3+json", "Authorization"=>"token 12345"}
irb(main):024:0> begin
irb(main):025:1*   f = Kernel.open("https://api.github.com/users", headers)
irb(main):026:1> rescue OpenURI::HTTPError => e
irb(main):027:1>   puts e.message
irb(main):028:1>   puts e.io.read
irb(main):029:1> end
401 Unauthorized
{"message":"Bad credentials","documentation_url":"https://developer.github.com/v3"}
```

`error` contains "401 Unauthorized"; the JSON is the same whether there's a bogus token or a bogus username/password so I think it isn't very compelling to do something else here.